### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by gamesminer**
task-1

### CODE REVIEW

**Comment by OlegShynkarenko** [comment]
Max line length should be 80 characters. 
Please check your index.html file and correct it manually or use js-beautify.

File: scss/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@font-face {
```
Comment (ihor4568) [inline scss/_variables.scss]: this is better to be extracted into separate fonts file

File: scss/components/_authors.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  &__list {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    align-items: center;
+    margin: 3.7rem auto 7rem;
+  }
+
+  &__item:first-child {
```
Comment (ihor4568) [inline scss/components/_authors.scss]: could you use modifiers/elements instead of nth-child?

File: scss/components/_button.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    background-color: #d2d5da;
+  }
+
+  &:hover {
+    opacity: 0.8;
+  }
+
+
+  @media only screen and (max-width: $smaller-break-point) {
+    & {
```
Comment (ihor4568) [inline scss/components/_button.scss]: no need for `& ` - you can write styled directly inside media

File: package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "gulp-sass": "4.0.1",
+    "husky": "^1.0.0-rc.14",
+    "js-beautify": "^1.8.6",
+    "lint-staged": "^7.2.2",
+    "node-sass": "4.9.3",
+    "prettier": "^1.14.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm run prettify-scss"
```
Comment (ihor4568) [inline package.json]: why do you need to run `prettify-scss` after lint-staged?

File: scss/components/_fa.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.fa {
```
Comment (ihor4568) [inline scss/components/_fa.scss]: `fa` is class used by font-awesome - you should not modify it

File: scss/components/_features.scss [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    width: 15rem;
+  }
+
+  &__heading_profesionals::before {
+    content: url(../img/icon_profesionals.png);
+    width: 39px;
+    height: 41px;
+    margin: 0 auto;
+    display: block;
+    padding-bottom: 3rem;
```
Comment (ihor4568) [inline scss/components/_features.scss]: why code is not formatted? are your precommit hooks working?

File: scss/components/_feedback.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+  &__subsection {
+    flex-basis: 350px;
+  }
+
+  &__heading {
+    padding-top: 0;
+  }
+
+  &__subsection:first-child {
```
Comment (ihor4568) [inline scss/components/_feedback.scss]: please use modifiers or elements instead of nth-child

File: scss/components/_feedback.scss [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    &__container {
+      flex-direction: column;
+    }
+
+    &__subsection:first-child,   &__subsection:nth-child(2) {
+      text-align: center;
+      margin: 5rem auto 0;
+    }
+
+  &__subsection:nth-child(2) {
```
Comment (ihor4568) [inline scss/components/_feedback.scss]: why code is not formatted?

File: scss/components/_header.scss [hunk 9]
```diff
@@ -0,0 +1,96 @@
+.header {
+  background: url('../img/bg.png') center/cover no-repeat;
+  color: white;
+
+  &__link_h1 {
```
Comment (ihor4568) [inline scss/components/_header.scss]: is this an element? or modifier? If modifier, then where is the element/block it modifies?

**Comment by ihor4568** [comment]
1 - Could you please attach screenshot of your site?
2 - you use `rem`s - and it is good - but I have not seen a place where you set base font size from which `rem`s will count their size

**Comment by gamesminer** [comment]
> 1 - Could you please attach screenshot of your site?
> 2 - you use `rem`s - and it is good - but I have not seen a place where you set base font size from which `rem`s will count their size

`  font: normal normal $basic-font-size $basic-font-stack; ` at '_page.scss'

File: package.json [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    "husky": "^1.0.0-rc.14",
+    "js-beautify": "^1.8.6",
+    "lint-staged": "^7.3.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```
[the remark this answers is not in the document]
Comment (gamesminer) [reply package.json]: Now starts

**Pull request merged**